modules/adminlog.cpp

```diff
@@ -85,6 +86,23 @@ class CAdminLogMod : public CModule {
 		Log("[" + sUsername + "] failed to login from " + sRemoteIP, LOG_WARNING);
 	}
 
+	void SetLogFilePath(CString& sPath) {
```

Comment:
Or, just CString. As it's modified below.
The point is, it shouldn't be modified on caller side.